bridges/snowbridge/pallets/system-frontend/src/lib.rs

```diff
@@ -40,26 +42,29 @@ use xcm_executor::traits::{FeeManager, FeeReason, TransactAsset};
 use frame_support::traits::OriginTrait;
 
 pub use pallet::*;
+pub type AccountIdOf<T> = <T as frame_system::Config>::AccountId;
 
 pub const LOG_TARGET: &str = "snowbridge-system-frontend";
 
 /// Call indices within BridgeHub runtime for dispatchables within `snowbridge-pallet-system-v2`
 #[allow(clippy::large_enum_variant)]
 #[derive(Encode, Decode, Debug, PartialEq, Clone, TypeInfo)]
-pub enum BridgeHubRuntime {
+pub enum BridgeHubRuntime<T: frame_system::Config> {
 	#[codec(index = 90)]
-	EthereumSystem(EthereumSystemCall),
+	EthereumSystem(EthereumSystemCall<T>),
 }
 
 /// Call indices for dispatchables within `snowbridge-pallet-system-v2`
 #[derive(Encode, Decode, Debug, PartialEq, Clone, TypeInfo)]
-pub enum EthereumSystemCall {
+pub enum EthereumSystemCall<T: frame_system::Config> {
 	#[codec(index = 0)]
 	RegisterToken {
 		sender: Box<VersionedLocation>,
 		asset_id: Box<VersionedLocation>,
 		metadata: AssetMetadata,
 	},
+	#[codec(index = 5)]
```

Comment:
Why is this index 5? You already had something in 1..4?

Reply:
Good catch, addressed in https://github.com/paritytech/polkadot-sdk/pull/8271/commits/d604ecc9e7ad2085ac0ab3a451e41132ad857e43.